{"uri":"file:///home/tarcisio/dev/phpfin/vendor/symfony/debug/Exception/ContextErrorException.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Symfony\\Component\\Debug\\Exception","location":[11,0,11,44]},{"kind":1,"name":"Symfony\\Component\\Debug\\Exception\\ContextErrorException","location":[18,0,35,1],"associated":[{"kind":1,"name":"ErrorException"}],"description":"Error Exception with Variable Context.","children":[{"kind":16,"name":"$context","modifiers":4,"location":[20,12,20,30],"scope":"Symfony\\Component\\Debug\\Exception\\ContextErrorException"},{"kind":32,"name":"__construct","modifiers":1,"location":[22,4,26,5],"scope":"Symfony\\Component\\Debug\\Exception\\ContextErrorException","children":[{"kind":128,"name":"$message","location":[22,32,22,40],"scope":"__construct"},{"kind":128,"name":"$code","location":[22,42,22,47],"scope":"__construct"},{"kind":128,"name":"$severity","location":[22,49,22,58],"scope":"__construct"},{"kind":128,"name":"$filename","location":[22,60,22,69],"scope":"__construct"},{"kind":128,"name":"$lineno","location":[22,71,22,78],"scope":"__construct"},{"kind":128,"name":"$context","location":[22,80,22,98],"scope":"__construct","value":"array()"}]},{"kind":32,"name":"getContext","modifiers":1,"location":[31,4,34,5],"scope":"Symfony\\Component\\Debug\\Exception\\ContextErrorException","type":"array"}]}]}}